src/main.rs

```diff
@@ -143,6 +138,65 @@ fn configure_color(cli_choice: Option<clap::ColorChoice>) {
     choice.write_global();
 }
 
+fn print_issue_url(config: &mut GlobalConfig) {
+    use bugreport::{bugreport, collector::*, format::Markdown};
+    let other_bug_url: &str = "https://github.com/obi1kenobi/cargo-semver-checks/issues/new?labels=C-bug&template=3-bug-report.yml";
+
+    let bug_report = bugreport!()
+        .info(SoftwareVersion::default())
+        .info(OperatingSystem::default())
+        .info(CommandLine::default())
+        .info(CommandOutput::new("cargo version", "cargo", &["-V"]))
+        .info(CompileTimeInformation::default())
```

Comment:
Is it possible to just `.clone()` here, so we can `.print()` once and `.format()` another time for the URL just to keep the code simpler?

Right now, we're mixing-and-matching formatting (e.g. on line 155), so if `bugreport` changes the output format in a future version, our output might look inconsistent.